make-release.sh

```diff
@@ -24,11 +23,11 @@ done
 
 usage ()
 {
-  echo "Usage: $0 --repo [GIT REPO TO EDIT] --version [VERSION TO RELEASE] [--trigger-release]"
-  echo "Example: $0 --repo git@github.com:eclipse/che-subproject --version 7.7.0 --trigger-release"; echo
```

Comment:
minor nitpick -- if you're updating this to remove the REPO variable, which we are no longer using (?) ... could also update the version in the example to something more current. Just a personal thing I do as a way of sort of versioning the code (ie., 7.26.0 shows this was updated way more recently than 7.7.0)